### PR TITLE
fix: enable development mode for the client

### PR DESCRIPTION
**Pull request by wtrocki**
Sample instructions for the latest release were broken. We were telling people to execute development commands instead of running the app. What is worse is that actual development was not possible due to missing config. 

I think we need to be more solid about templates testing as this was broken for an month without anyone using/noticing.

### CODE REVIEW

**Comment by wtrocki** [comment]
Issue reported by architect.

File: templates/ts-apollo-mongodb-backend/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@types/glob": "7.1.3",
     "@types/graphql": "14.2.3",
     "@types/node-fetch": "2.5.7",
-    "graphback-cli": "0.15.1",
+    "graphback-cli": "0.16.0-beta3",
```
Comment (machi1990) [inline templates/ts-apollo-mongodb-backend/package.json:33]: Should this stay as was or changed or the latest release? [0.16.0-beta4](https://github.com/aerogear/graphback/releases/tag/0.16.0-beta4)?

File: templates/ts-react-apollo-client/package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "@graphql-codegen/typescript": "1.17.9",
     "@graphql-codegen/typescript-operations": "1.17.8",
     "@graphql-codegen/typescript-react-apollo": "2.0.6",
-    "graphback-cli": "0.15.1",
+    "graphback-cli": "0.16.0-beta2",
```
Comment (machi1990) [inline templates/ts-react-apollo-client/package.json:11]: See https://github.com/aerogear/graphback/pull/1957/files#r478640431

**Comment by machi1990** [comment]
Thanks for catching this, I've added some comments. 

> I think we need to be more solid about templates testing as this was broken for an month without anyone using/noticing.

Yes, I think the templates could use some smoke tests.

File: templates/ts-apollo-mongodb-backend/package.json (hunk 1 above)
Comment (wtrocki) [inline templates/ts-apollo-mongodb-backend/package.json:33]:
```suggestion
    "graphback-cli": "0.15.1",
```

**Review by craicoverflow — CHANGES_REQUESTED** [review]

File: templates/ts-react-apollo-client/package.json (hunk 2 above)
Comment (wtrocki) [inline templates/ts-react-apollo-client/package.json:11]:
```suggestion
    "graphback-cli": "0.15.1",
```

**Review by machi1990 — APPROVED** [review]
Comments addressed and this is good for me. 

@craicoverflow do you want to have a look? It appears to me that your comment has been addressed too.

**Review by craicoverflow — APPROVED** [review]

**Comment by wtrocki** [comment]
Thx guys! Let's include this into 0.16.0 release

**Pull request merged**